src/Component/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "./ProductDetail.scss";
+import "../../Styles/reset.scss";
+import "../../Styles/common.scss";
+import Header from "./Component/Header/Header";
+import ProductReview from "./Component/Article/ProductReview/ProductReview";
+import ProductInfo from "./Component/Article/ProductInfo/ProductInfo";
+
+class ProductDetail extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productdetail: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/productdetail.json", {
+      method: "GET",
+    })
```

Comment:
- localhost는 포트번호가 달라질 수 있기 떄문에 삭제하시는게 좋습니다. 
- get method는 default 값이기 때문에 삭제할 수 있습니다. 

```suggestion
    fetch("/data/productdetail.json")
```

Reply:
저번에 알려주신, config에 넣는 방법을 사용해 보려고 하였으나 적용이 되지 않더라구요... 무슨 문제일까요??